internal/support-jira-announcement-tool/send-announcements.sh

```diff
@@ -34,6 +34,16 @@ read log_file
 #compares the skip list & the project list and makes a filtered list
 diff $proj_file $skip_file |  grep  '<' | sed 's/< //g' > filtered
 
+#Creates a file called 'all_filtered' by removing the closed projects from the filtered file
+while IFS="," read -r id key description remainder
+do
+	if [ "$description" == '"Development Support"' ] || [ "$description" == '"Evaluation Support"' ] || [ "$description" == '"High-Value Evaluation"' ] || [ "$description" == '"Managed Cloud Development Support"' ] || [ "$description" == '"Managed Cloud Hosting Support"' ] || [ "$description" == '"Managed Cloud Production Support"' ] || [ "$description" == '"Managed Cloud Subscription"' ] || [ "$description" == '"Production Support"' ] || [ "$description" == '"Subscription"' ]
```

Comment:
Can't we define an array that contains these values and then iterate the description variable against the array? 

Example in [1]

[1] https://unix.stackexchange.com/questions/177138/how-do-i-test-if-an-item-is-in-a-bash-array